From 7757561f61ea8e8319e8aa4e5b5aa9d40e8d0bc1 Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:36:59 -0400
Subject: [ethernet] i40e: Use DEBUG_FD message level for an FD message

Message-id: <1409658042-22689-59-git-send-email-sassmann@redhat.com>
Patchwork-id: 90252
O-Subject: [RHEL7.1 PATCH 058/281] i40e: Use DEBUG_FD message level for an FD message
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Anjali Singhai Jain <anjali.singhai@intel.com>

We don't need to print this info unless at FD message level.

Change-ID: I329efdd8e754a0ea0669ec04d12e03db02e6b76e
Signed-off-by: Anjali Singhai Jain <anjali.singhai@intel.com>
Signed-off-by: Catherine Sullivan <catherine.sullivan@intel.com>
Tested-by: Kavindya Deegala <kavindya.s.deegala@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit 13c2884f155bc524c5e94482216030de480fea60)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_txrx.c b/drivers/net/ethernet/intel/i40e/i40e_txrx.c
index de25a32..a329aac 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_txrx.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_txrx.c
@@ -482,8 +482,9 @@ static void i40e_fd_handle_status(struct i40e_ring *rx_ring,
   }
  } else if (error ==
      (0x1 << I40E_RX_PROG_STATUS_DESC_NO_FD_ENTRY_SHIFT)) {
-  netdev_info(rx_ring->vsi->netdev, "ntuple filter loc = %d, could not be removed\n",
-       rx_desc->wb.qword0.hi_dword.fd_id);
+  if (I40E_DEBUG_FD & pf->hw.debug_mask)
+   dev_info(&pdev->dev, "ntuple filter loc = %d, could not be removed\n",
+     rx_desc->wb.qword0.hi_dword.fd_id);
  }
 }
 
-- 
1.7.1